.github/workflows/deploy-admin.yml

```
@@ -51,26 +51,11 @@ jobs:
         run: |
-          sed -i "s|image: .*|image: ${{ secrets.DOCKER_USERNAME }}/admin-service:${SHORT_SHA}|" admin/k8s/deployment.yml
-          kubectl --kubeconfig=kubeconfig.yaml apply -f admin/k8s/
-          kubectl --kubeconfig=kubeconfig.yaml rollout status deployment admin-service -n onevoice --timeout=60s
-        env:
-          KUBECONFIG: ${{ github.workspace }}/kubeconfig.yaml
+          sed -i "s|newTag: .*|newTag: ${SHORT_SHA}|" admin/k8s/overlays/prod/kustomization.yml
+          git config user.name "github-actions"
+          git config user.email "actions@github.com"
+          git add admin/k8s/overlays/prod/kustomization.yml
+          git commit -m "chore: update admin-service tag to ${SHORT_SHA}"
+          git push origin develop
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)